doc/src/redis.md

```diff
@@ -20,8 +20,9 @@ In previous versions, custom redis configuration could be set
 via {file}`/etc/local/redis/custom.conf` which is not supported anymore.
 
 If you need to change the behaviour of Redis, you define your redis
-configuration with the NixOS option `services.redis.settings`. See the
-NixOS manual for further information.
+configuration with the NixOS option
+`services.redis.servers."".settings`. See the NixOS manual for further
+information.
 
```

Comment:
It might be worth explaining the unusual option name with the empty instance string here, at least briefly. Otherwise readers might deem this to be a typo.

Reply:
Something like this?
```
If you need to change the behaviour of Redis, you define your Redis
configuration with the NixOS option
`services.redis.servers."".settings`. NixOS supports multiple
instances of Redis on a single host, so this option sets configuration
for the default instance with an empty instance name. See the NixOS
manual for further information.
```